cli/command/stack/deploy.go

```diff
@@ -81,7 +81,8 @@ func RunDeploy(dockerCli command.Cli, flags *pflag.FlagSet, config *composetypes
 	case commonOrchestrator.HasKubernetes():
 		kli, err := kubernetes.WrapCli(dockerCli, kubernetes.NewOptions(flags, commonOrchestrator))
 		if err != nil {
-			return err
+
```

Comment:
Oh, erm, can I be nit-picky? Could you remove this empty line?

Reply:
Sure can! Will do that now.